[2025-04-05 16:22:08.856] [Xeno] [info] Version: Xeno - v1.1.7
[2025-04-05 16:22:08.856] [Xeno] [info] Logs stored in logs/2025-04-05_16-22-08.log
[2025-04-05 16:22:08.857] [Xeno] [debug] Client Checker thread: 0x555757d3e0
[2025-04-05 16:22:08.857] [Xeno] [debug] Xeno Caching thread: 0x555757d3d0
[2025-04-05 16:22:08.857] [Xeno] [debug] WebSockets Cleaner thread: 0x555757d3c0
[2025-04-05 16:22:08.857] [Cache] [info] File: C:\Users\bp210\AppData\Local\Temp\XENO_CACHE.bin
[2025-04-05 16:22:08.858] [Xeno] [debug] Xeno Server thread: 0x555757d3b0
[2025-04-05 16:22:08.858] [Cache] [info] Reattaching to cached client PID: 22916
[2025-04-05 16:22:08.858] [Cache] [info] Loaded cache: 1 RBXClients, 0 RBXBytecode, 0 RBXMFlags
[2025-04-05 16:22:08.858] [Xeno] [info] Creating new RBXClient: 22916
[2025-04-05 16:22:08.858] [RBXClient] [info] [22916]: Initializing...
[2025-04-05 16:22:08.858] [Server] [info] Listening on http://localhost:3110
[2025-04-05 16:22:08.858] [RBXClient] [info] [22916]: Version: version-347f4ac346734391
[2025-04-05 16:22:08.858] [Xeno] [debug] Created RBXClient instance @0x243825167f0
[2025-04-05 16:22:08.859] [Xeno] [info] Successfully created RBXClient: 22916
[2025-04-05 16:22:08.860] [RBXClient] [debug] [22916]: Getting RenderView...
[2025-04-05 16:22:08.874] [Xeno] [info] Terminated existing Xeno process: 31928
[2025-04-05 16:22:08.986] [RBXClient] [debug] [22916]: String: 0x21118b83d90
[2025-04-05 16:22:08.986] [RBXClient] [debug] [22916]: Renderview: 0x21173d1e230
[2025-04-05 16:22:08.986] [RBXClient] [debug] [22916]: DataModel: 0x2116bc03290
[2025-04-05 16:22:08.994] [RBXClient] [debug] [22916]: LocalPlayer: 0x211701632f0
[2025-04-05 16:22:08.994] [RBXClient] [debug] [22916]: LocalPlayer Name: 'lcg93'
[2025-04-05 16:22:09.291] [RBXClient] [debug] [22916]: EventConnectionEmbedded: 0x211376b0c10
[2025-04-05 16:22:09.292] [RBXClient] [info] [22916]: Bytecode has been successfully set in menu.
[2025-04-05 16:22:31.470] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:22:35.714] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:22:35.747] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:22:36.980] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:22:37.013] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:30:02.846] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:16.022] [WebSocket] [debug] Connection established for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:16.257] [WebSocket] [debug] Connection established for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:32.949] [WebSocket] [debug] Connection established for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:48.779] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:48.860] [WebSocket] [debug] Connection established for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:30:52.370] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:30:52.392] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:30:53.624] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:30:53.656] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:31:09.852] [WebSocket] [debug] WebSocket Closed for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:31:10.271] [WebSocket] [debug] WebSocket Closed for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:31:20.285] [WebSocket] [debug] WebSocket Closed for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:31:36.060] [WebSocket] [debug] WebSocket Closed for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:32:35.086] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:32:44.418] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:32:48.848] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:32:48.893] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:32:50.346] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:32:50.388] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:33:22.105] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:33:47.497] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:33:50.736] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:33:50.770] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:33:52.001] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:33:52.035] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:36:29.562] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:36:56.593] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:37:00.830] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:37:00.863] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:37:02.180] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:37:02.213] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:37:29.844] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:37:37.284] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:38:05.499] [WebSocket] [debug] Connection established for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:38:09.909] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:38:09.942] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:38:11.177] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:38:11.208] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:38:29.369] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:43:27.663] [WebSocket] [debug] WebSocket Closed for 'wss://as2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:43:36.745] [WebSocket] [debug] Connection established for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:43:41.281] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:43:41.341] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:43:42.916] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 16:43:42.949] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 16:43:44.818] [WebSocket] [debug] Connection established for 'wss://au1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 16:44:35.545] [WebSocket] [debug] WebSocket Closed for 'wss://au1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:00:04.395] [WebSocket] [debug] WebSocket Closed for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:00:13.699] [WebSocket] [debug] Connection established for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:00:16.646] [WebSocket] [debug] Connection established for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:00:20.573] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:00:20.607] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:00:21.839] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:00:21.873] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:01:06.617] [WebSocket] [debug] WebSocket Closed for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:01:31.403] [WebSocket] [debug] WebSocket Closed for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:01:43.568] [WebSocket] [debug] Connection established for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:01:47.520] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:01:47.553] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:01:48.786] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:01:48.819] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:02:17.169] [WebSocket] [debug] WebSocket Closed for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:02:24.641] [WebSocket] [debug] Connection established for 'wss://au1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:03:12.044] [WebSocket] [debug] Connection established for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:03:15.992] [WebSocket] [debug] WebSocket Closed for 'wss://au1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:03:16.400] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:03:16.434] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:03:17.668] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:03:17.702] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:04:16.131] [WebSocket] [debug] WebSocket Closed for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:04:28.349] [WebSocket] [debug] Connection established for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:04:34.547] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:04:34.580] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:04:35.814] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:04:35.847] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:04:58.594] [WebSocket] [debug] WebSocket Closed for 'wss://au2-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:05:18.915] [WebSocket] [debug] Connection established for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:05:23.295] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:05:23.329] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:05:24.562] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC"
[2025-04-05 17:05:24.595] [Server] [error] Encountered an exception: create_directory: unknown error: "Lunor/AC/settings"
[2025-04-05 17:06:27.392] [WebSocket] [debug] WebSocket Closed for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:06:35.037] [WebSocket] [debug] Connection established for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:07:08.330] [WebSocket] [debug] Connection established for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:07:20.553] [WebSocket] [debug] WebSocket Closed for 'wss://as1-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:08:00.964] [WebSocket] [debug] WebSocket Closed for 'wss://as3-roblox-auth.luarmor.net:443/wshttpemu'
[2025-04-05 17:08:19.644] [Xeno] [info] Removed RBXClient: 22916
